content/contributing/helping-out/reviewing-pull-requests.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Reviewing Pull Requests"
+insert_anchor_links = "right"
+[extra]
+weight = 4
++++
+
+With the sheer volume of activity in Bevy's community, reviewing others' work with the aim of improving it is one of the most valuable things you can do.
+
+## Who can review
+
+Anyone!
+
+You don't need to be an Elder Rustacean to be useful here: anyone can catch missing tests, unclear docs, logic errors, and so on. If you have specific skills (e.g. advanced familiarity with unsafe code, rendering knowledge or web development experience) or personal experience with a problem, try to prioritize those areas to ensure we can get appropriate expertise where we need it.
+
+If you find a PR that you don't feel comfortable reviewing, but you can think of someone who does, consider using Github's "Request review" functionality (in the top-right of the PR screen) to bring the work to their attention. If they're not a Bevy Org member, you'll need to ping them in the thread directly: that's fine too! Almost everyone working on Bevy is a volunteer: this should be treated as a gentle nudge, rather than an assignment of work. Consider checking the Git history for appropriate reviewers, or ask on Discord for suggestions.
+
+## How to review a Pull Request
+
+If you're new to GitHub, there's a lot of great information on the official [Pull Request Review documentation](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews). If you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your Approved review on the _PR_. Again, anyone can and should leave reviews ... no special permissions are required!
```

Comment:
```suggestion
If you're new to GitHub, there's a lot of great information on the official [Pull Request Review documentation](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews). If you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your Approved review on the *PR*. Again, anyone can and should leave review— no special permissions are required!
```
Change the underscores to asterisks to have consistent italics style. The em-dash should work better as an interruption in this context.